drivers/can/can_sam0.c

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright (c) 2022 Vestas Wind Systems A/S
+ * Copyright (c) 2021 Alexander Wachter
+ * Copyright (c) 2022 Kamil Serwus
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/drivers/can.h>
+#include <zephyr/drivers/pinctrl.h>
+#include <soc.h>
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
```

Comment:
Could you please also add `#include <zephyr/irq.h>` as you are using `IRQ_CONNECT`